maven-plugin/src/main/java/org/openmrs/maven/plugins/bintray/OpenmrsBintray.java

```diff
@@ -60,4 +70,30 @@ public void extractOwa(File owaZip){
             throw new RuntimeException(e);
         }
     }
+
+    public BintrayPackage getMavenPackageMetadata(String name) {
+        return getPackageMetadata(OPENMRS_USERNAME, OPENMRS_MAVEN_REPO, name);
+    }
+
+    public List<BintrayId> getMavenAvailablePackages(String repo) {
+        return getAvailablePackages(OPENMRS_USERNAME, OPENMRS_MAVEN_REPO);
+    }
+
+    public BintrayPackage createMavenPackage(MavenProject mavenProject){
+        CreatePackageRequest request = new CreatePackageRequest();
+        request.setName(mavenProject.getArtifactId());
+        request.setDescription(mavenProject.getDescription());
+
+        String githubUrl = mavenProject.getScm().getUrl();
+        String githubRepo = githubUrl.substring(githubUrl.lastIndexOf("/"));
+        request.setVcsUrl(githubUrl+".git");
+        request.setGithubRepo(OPENMRS_USERNAME+githubRepo);
+        request.setWebsiteUrl("http://openmrs.org/");
+
+        request.setIssueTrackerUrl(new DefaultJira().getJiraUrl());
+        //so far all OpenMRS projects have MPL-2.0 license
+        //TODO: resolve it dynamically?
+        request.setLicenses(Arrays.asList("MPL-2.0"));
```

Comment:
@rkorytkowski what do you think about this? is hard-coded MPL license sufficient?